voice2text_deltabot/hooks.py

```diff
@@ -12,7 +12,7 @@
 cli = BotCli("voice2text-bot")
 add_subcommands(cli)
 STATUS = "I am a Delta Chat bot, send me any voice message to convert it to text"
-MODEL: whisper.Whisper = None  # noqa
+MODEL: WhisperModel = None # noqa
```

Comment:
the comment must have 2 spaces before the #, you should use black to format the code